homeassistant/components/airtouch4/climate.py

```diff
@@ -294,9 +295,11 @@ def fan_modes(self):
         )
         return [AT_TO_HA_FAN_SPEED[speed] for speed in airtouch_fan_speeds]
 
-    async def async_set_temperature(self, **kwargs):
+    async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperatures."""
-        temp = kwargs.get(ATTR_TEMPERATURE)
+        if (temp := kwargs.get(ATTR_TEMPERATURE)) is None:
```

Comment:
`ATTR_TEMPERATURE` is not always present in `kwargs`, but mypy is rejecting `int(temp)` as `temp` can be `None`.
This ensure that temp is not None before sending the request.